go.mod

```
@@ -3,6 +3,7 @@ module github.com/xuri/excelize/v2
 go 1.23.0
 
 require (
+	github.com/mohae/deepcopy v0.0.0-20170929034955-c48cc78d4826
 	github.com/richardlehane/mscfb v1.0.4
 	github.com/stretchr/testify v1.10.0
 	github.com/tiendc/go-deepcopy v1.5.1
```

```suggestion
	github.com/tiendc/go-deepcopy v1.5.2
```

The bug for `sync.Map` has been fixed in v1.5.2. https://github.com/tiendc/go-deepcopy/issues/34#issuecomment-2829145854

https://go.dev/play/p/cyQ8e6Zh-mX

---

Thank you for the update. Upgrade the package to keep using `go-deepcopy` instead of introduce new package `deepcopy`.

---

I update this pull request, upgrade package go-deepcopy to v1.5.2.